prometheus/build.gradle

```diff
@@ -9,6 +9,11 @@ plugins {
     id 'jacoco'
 }
 
+repositories {
+    mavenCentral()
+    maven { url 'https://jitpack.io' }
+}
+
 dependencies {
     api project(':core')
     api group: 'org.opensearch', name: 'opensearch', version: "${opensearch_version}"
```

Comment:
why prometheus depend on opensearch?

Reply:
Thanks. Got copied over...will remove all the opensearch dependencies and see if they are really required.